### ISSUE TITLE
SVG still not fully rendered

### ISSUE THREAD

**Issue by peterennis** [ionic-team/stencil#882]
**Stencil version:**

```
C:\ae\adaept.com\ae-logo-component>npm list @stencil/core
ae-logo-component@0.0.4 C:\ae\adaept.com\ae-logo-component
`-- @stencil/core@0.9.11
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

**Current behavior:**
I want to generate SVG output but JSX does not want to cooperate
or there is some secret foo missing.

**Expected behavior:**
SVG should be rendered

**Steps to reproduce:**

**Related code:**
https://github.com/peterennis/ae-logo-component

These commits:
https://github.com/peterennis/ae-logo-component/commit/dcac47978beda68522fd9573d49a7e23543bf8cb
https://github.com/peterennis/ae-logo-component/commit/713f93ec2917be4943565367a7782a67c27719d7

**Other information:**
Debug output.
![capture329](https://user-images.githubusercontent.com/140737/41889286-9c6f1b94-78be-11e8-8201-b948eb40f0e3.png)

Related bug: https://github.com/ionic-team/stencil/issues/448

**Comment by peterennis** [comment]
Same problem.
```
C:\ae\adaept.com\ae-logo-component>npm list @stencil/core
ae-logo-component@0.0.4 C:\ae\adaept.com\ae-logo-component
`-- @stencil/core@0.10.0-3
```

**Comment by peterennis** [comment]
Closed as duplicate.
Fixed in #448
